cardano_node_tests/tests/tests_conway/test_committee.py

```diff
@@ -104,101 +104,54 @@ class TestCommittee:
     @pytest.mark.dbsync
     @pytest.mark.testnets
     @pytest.mark.smoke
-    @pytest.mark.disabled(reason="Needs to be fixed in 9.0.0+")
-    def test_register_and_resign_committee_member(
+    def test_resign_committee_member(
         self,
-        cluster_use_committee: governance_utils.GovClusterT,
-        payment_addr_comm: clusterlib.AddressRecord,
+        cluster_lock_governance: governance_utils.GovClusterT,
+        pool_user_lg: clusterlib.PoolUser,
         use_build_cmd: bool,
         submit_method: str,
     ):
-        """Test Constitutional Committee Member registration and resignation.
+        """Test Constitutional Committee Member resignation.
 
-        * register a potential CC Member
-        * check that CC Member was registered
         * resign from CC Member position
         * check that CC Member resigned
         """
-        cluster, __ = cluster_use_committee
+        cluster, governance_data = cluster_lock_governance
         temp_template = common.get_test_id(cluster)
 
-        # Register a potential CC Member
+        # Resignation of CC Member
 
         _url = helpers.get_vcs_link()
-        [
-            r.start(url=_url)
-            for r in (reqc.cli003, reqc.cli004, reqc.cli005, reqc.cli006, reqc.cip003)
-        ]
-        cc_auth_record = governance_utils.get_cc_member_auth_record(
-            cluster_obj=cluster,
-            name_template=temp_template,
-        )
-        [r.success() for r in (reqc.cli003, reqc.cli004, reqc.cli005, reqc.cli006)]
-
-        tx_files_auth = clusterlib.TxFiles(
-            certificate_files=[cc_auth_record.auth_cert],
-            signing_key_files=[payment_addr_comm.skey_file, cc_auth_record.cold_key_pair.skey_file],
-        )
-
-        tx_output_auth = clusterlib_utils.build_and_submit_tx(
-            cluster_obj=cluster,
-            name_template=f"{temp_template}_auth",
-            src_address=payment_addr_comm.address,
-            submit_method=submit_method,
-            use_build_cmd=use_build_cmd,
-            tx_files=tx_files_auth,
-        )
-        reqc.cip003.success()
-
-        auth_out_utxos = cluster.g_query.get_utxo(tx_raw_output=tx_output_auth)
-        assert (
-            clusterlib.filter_utxos(utxos=auth_out_utxos, address=payment_addr_comm.address)[
-                0
-            ].amount
-            == clusterlib.calculate_utxos_balance(tx_output_auth.txins) - tx_output_auth.fee
-        ), f"Incorrect balance for source address `{payment_addr_comm.address}`"
+        [r.start(url=_url) for r in (reqc.cli007, reqc.cip012)]
 
-        cluster.wait_for_new_block(new_blocks=2)
-        _url = helpers.get_vcs_link()
-        [r.start(url=_url) for r in (reqc.cli032, reqc.cip002, reqc.cip004)]
```

Comment:
The coverage of reqc.cli032, reqc.cip002, reqc.cip004 is removed, and not added to some other test.